docs/src/guides/client_configuration.md

```diff
@@ -12,7 +12,7 @@ Here is an example of the `LSP.sublime-settings` file with configurations for th
 ```js
 {
   // General settings
-  "log_stderr": true,
+  "log_debug": true,
```

Comment:
Should we suggest this one really?

Reply:
I could put `show_view_status` instead of it, or some other setting?

Reply:
`"show_diagnostics_panel_on_save": 0` is IMO a good example

Reply:
b1fefb1